.gitpod.yml

```diff
@@ -0,0 +1,30 @@
+github:
+  prebuilds:
+    master: true
+    branches: true
+    pullRequests: true
+    pullRequestsFromForks: true
+    addCheck: false
+    addComment: false
+    addBadge: true
+
+tasks:
+  - name: Start web server
+    command: python3 -m http.server --directory examples/wasm
+
+  - name: Install linux dependencies
+    before: |
+      sudo apt-get install --no-install-recommends -yq libasound2-dev libudev-dev
```

Comment:
Since we're building for wasm, why is this needed?

Reply:
for rust-analyzer. I guess I could add config to have it work in wasm by default, but I'm not sure where that would live

Reply:
That is rust-analyzer.cargo.target at wherever gitpod allow configuring rust-analyzer. For vscode that would be .vscode/settings.json.

Reply:
I *think* I added the file correctly, but still keeping the linux dependencies because they add almost nothing to the build time and they are still helpful